src/main/kotlin/com/launchdarkly/intellij/settings/LaunchDarklyApplicationSettings.kt

```diff
@@ -237,8 +237,8 @@ class LaunchDarklyApplicationConfigurable : BoundConfigurable(displayName = "Lau
             projectContainer = getProjects(String(accessTokenField.password), settings.baseUri)
             with(projectBox) {
                 removeAllElements()
-                selectedItem = projectContainer.map { it.key }.firstOrNull()
-                projectContainer.map { addElement(it.key) }
+                selectedItem = projectContainer.map { it.key.plus(" (").plus(it.name).plus(")") }.firstOrNull()
```

Comment:
can `it.name` be null or empty?

Reply:
My understanding is that in kotlin it's difficult to have a null variable ([see this article](https://kotlinlang.org/docs/null-safety.html#checking-for-null-in-conditions)), however a null check couldn't hurt so I can add that.

Reply:
yeah, I think that's a fair point, I think if the language doesn't allow for it, we don't have to overdue it with null checks, my concern would be an empty string then because I don't want to end up with this `production ()`, can we update the format helper method to not append the `($safeName)` if `name` is an empty string?

Reply:
Updated the format method to only print key if name is null or empty